src/utilities/utils.ts

```diff
@@ -198,3 +199,12 @@ export function generateRandomString(length: int): string {
 
     return randomString;
 }
+
+export function seededShuffle<T>(list: T[], seed: string): T[] {
+    const random: seedrandom.PRNG = seedrandom(seed);
```

Comment:
Does const random hold a function?
In this way, given a string seed, random() generates the same sequence of number no matter which client calls it?

Reply:
Yess correct!